challenges/generic-type-arguments/prompt.md

```diff
@@ -0,0 +1,95 @@
+## Why We Need Generic Type Arguments
+
+If primitive (and literal) types are data, you can think of generics as functions that operate on that data.
+
+And just like functions often need arguments, we need some way to provide inputs to generic types.
+
+## How To Use Generic Arguments
+
+Instead of using parenthesis to pass arguments, with generics we use angled brackets: `<` and `>`.
```

Comment:
Should we mention that it's *kinda* similar to generics in other languages (Java, C#, Rust...)?

Reply:
ehh.  personally I don't know what value it adds.  there are enough that don't anyway (Go, Carbon, Haskell, etc.).  but I don't feel strongly at all about not adding it.  You have push access on my fork, please (really, seriously) feel absolutely totally free to push this if you'd like.